reference-implementation/lib/helpers.js

```diff
@@ -62,6 +62,10 @@ exports.IsFiniteNonNegativeNumber = v => {
 };
 
 exports.InvokeOrNoop = (O, P, args) => {
+  assert(exports.typeIsObject(O) === true);
+  assert(typeof P === 'string');
```

Comment:
Yeah, should probably do that, but then again, all callers are guaranteed to pass a constant string, so do we need to allow Symbols if they're never used?